src/App/pages/LandingPage/Main/FilterFormCinema/FilterFormCinema.js

```diff
@@ -0,0 +1,35 @@
+export function FilterFormCinema({}) {
+    let cinemaData =[{"id":655,"cinemaName":"eiufeiufheuf","adress":{"streetAddress":"6388 Lacus Ct","city":"Kenora"}},
+    {"id":761,"cinemaName":"qsqsq","adress":{"streetAddress":"6057 Vitae Rd","city":"Washington"},},
+    {"id":764,"cinemaName":"wdwdwdsq","adress":{"streetAddress":"5435 Sollicitudin Ln","city":"Hudsonville"},},
```

Comment:
You also have a type in the 'adress'. It is supposed to be 'address'.

Reply:
fixed